wgpu-core/Cargo.toml

```diff
@@ -30,7 +30,8 @@ codespan-reporting = "0.11"
 copyless = "0.1"
 fxhash = "0.2"
 log = "0.4"
-parking_lot = "0.12"
+# parking_lot 0.12 switches from `winapi` to `windows`; permit either
+parking_lot = ">=0.11,<=0.12"
```

Comment:
I might be wrong here, but I think we need this to be <0.13 as 0.12.1 is technically greater than 0.12

Reply:
Yes - thanks.